library/src/main/java/com/novoda/downloadmanager/MigrationServiceBinderBuilder.java

```diff
@@ -0,0 +1,70 @@
+package com.novoda.downloadmanager;
+
+import android.content.Context;
+import android.os.Handler;
+
+public class MigrationServiceBinderBuilder {
```

Comment:
Provides defaults for the `Binder` and wraps the callback on a main thread handler, or any handler that the client specifies.